cmd/tsdbrelay/main.go

```diff
@@ -14,6 +15,8 @@ import (
 	"os"
 	"strings"
 
+	version "bosun.org/_version"
```

Comment:
I can't explain why. If it helps to remove the alias, go ahead, but it sounds like something odd may be going on with your go environment.